mix.lock

```diff
@@ -1,5 +1,6 @@
 %{
   "atecc508a": {:hex, :atecc508a, "0.2.2", "9f52e587f43b658e8a4dd1dadbe5cbad9b00d17001e6f47fd9198cc6ce4d8245", [:mix], [{:circuits_i2c, "~> 0.2", [hex: :circuits_i2c, repo: "hexpm", optional: false]}, {:x509, "~> 0.5.1 or ~> 0.6", [hex: :x509, repo: "hexpm", optional: false]}], "hexpm", "d268892f39763957945a5b3ded537bcdffe323486e05cfac3cde0f026afb32c1"},
+  "castore": {:hex, :castore, "0.1.9", "eb08a94c12ebff92a92d844c6ccd90728dc7662aab9bdc8b3b785ba653c499d5", [:mix], [], "hexpm", "99c3a38ad9c0bab03fee1418c98390da1a31f3b85e317db5840d51a1443d26c8"},
```

Comment:
Is `castore` pulled in because of `mint`?

Reply:
correct, it's required for SSL requests